sovrin_client/agent/walleted.py

```diff
@@ -97,6 +103,17 @@ def __init__(self,
         }
         self.logger = logger
 
+        self.issuer_backend = None
+
+        self._invites = {}  # type: Dict[Nonce, Tuple(InternalId, str)]
+        self._attribDefs = {}  # type: Dict[str, AttribDef]
+        self.defined_claims = []  # type: List[Dict[str, Any]
+
+        self.available_claims_by_internal_id = {}  # type: Dict[InternalId, Set[ID]]
```

Comment:
actually remembered one more thing which would be helpful for you, so far, on master branch code, most of the states (links/connections, attributes, ids etc) were in wallet.py class, and whenever we had to persist wallet (either for client (like Alice etc) or for agent (like Faber etc)), we used to just persist that wallet object and when client/agent used to restart, that persisted wallet used to restored and thats how things were working so far. Now I see good amount of things are getting created/shifted in walleted.py which we are not persisting, that will break things, like clients and agents won't have the same state after restart and it will break few things I guess. What is the reason to keep them in walleted.py instead of wallet.py?